packages/run-protocol/src/vaultFactory/addAssetToVault.js

```diff
@@ -130,14 +130,15 @@ export const addAssetToVault = async ({
   });
 };
 
-export const getManifestForStep2 = (
+export const getManifestForAddAssetToVault = (
   { restoreRef },
-  { denom, scaledPriceAuthority },
+  { denom, scaledPriceAuthorityRef },
 ) => {
   return {
     manifest: {
       addInterchainAsset: {
         consume: { zoe: true, bankManager: true, agoricNamesAdmin: true },
+        produce: { ibcAtomMintForTesting: true },
```

Comment:
Passing this mint out for testing is an expedient hack. hm.

Reply:
I kept the mints around even not when testing because I don't want to drop anything that valuable on the floor.